zebra-state/src/service/finalized_state/zebra_db/transparent.rs

```diff
@@ -235,44 +235,25 @@ impl ZebraDb {
         let tx_loc_by_transparent_addr_loc =
             self.db.cf_handle("tx_loc_by_transparent_addr_loc").unwrap();
 
-        // Manually fetch the entire addresses' transaction locations
-        let mut addr_transactions = BTreeSet::new();
-
         // A potentially invalid key representing the first UTXO send to the address,
         // or the query start height.
-        let mut transaction_location = AddressTransaction::address_iterator_start(
+        let transaction_location = AddressTransaction::address_iterator_start(
```

Comment:
Optional Nitpick: since we're no longer modifying this variable, it is always the `first_transaction_location`.